### ISSUE TITLE
Raven RStudio keeps crashing

### ISSUE THREAD

**Issue by valeste** [course-fish546-2023/assistance-public#6]
For the past two days I have been unable to use Raven's RStudio Server as the page continues to be unresponsive each time I login. The page eventually crashes and produces the ' RESULT_CODE_HUNG' error. I have already tried the following suggestions but none have helped:

- Cleared cached files and cookies from browsing history on Google Chrome
- Restarted computer
- Restarted Google Chrome
- Tried using the server through an incognito window on Google Chrome
- Tried using a different web browser (Safari)

Has anyone else experienced this issue before and are there any other work arounds aside from what I have already tried to remedy this?

Also, when I SSH into the server from a local terminal and run the following command:

```
du -h --max-depth=1
```
This is the size of my remote project directory on Raven. Could the size of the directory be related to the issue I am experiencing?

16G     ./.git
12K     ./notes
1.3G    ./data
1.7G    ./output
27M     ./.Rproj.user
11M     ./code
19G     .

Thanks!

**Comment by sr320** [comment]
While sometime slow I have been using past two days.  
I just killed your `rsession` on server so try again now in event something you were doing got hung.

Also you could ssh in and run code on terminal, (or use mox, or jupyterlab)

I do not think the size of your files plays a role 
here is mine

```
(base) sr320@raven:/home/shared/8TB_HDD_01/sr320/github/steven-crabby-duck$ du -h --max-depth=1
4.0M    ./code
744K    ./.Rproj.user
28G     ./data
227G    ./output
81G     ./.git
336G    .
```

**Comment by valeste** [comment]
Thanks for killing my rsession, I suspected that it got stuck running some job, but that didn't seem to help. I suppose I will shh in and run my code on the terminal for now. Maybe the issue will resolve itself later on?

**Comment by sr320** [comment]
Is the responsive different on / off campus?

**Comment by valeste** [comment]
Yes, the server page is unresponsive both using the Husky OnNet VPN and on (UW Tacoma) campus. I haven't been to the Seattle campus so I am not sure if it is unresponsive also there.

**Comment by valeste** [comment]
@kubu4

**Comment by kubu4** [comment]
@sr320 - Which command(s) did you use to kill @valeste's R session?

**Comment by sr320** [comment]
`kill -9 PID`

**Comment by kubu4** [comment]
@valeste - You should be able to log in now without the crashing.

After a bunch of different command line junk which didn't accomplish anything, I managed to close your open project (`celeste-tunicate-devo`) using some quick mouse clicking and touchscreen dexterity!

However, when this project is loaded, it causes the same hanging issue.

So, my advice is to delete this project/repo and start a new one. 

Since the project was hanging immediately, it's hard to say what was causing the issue. I did manage to clear the environment at one point, but the issue persisted, so it wasn't related to large objects or something being stored in memory.

In any case, delete the old project and create a new one.

Let me know if you need any assistance with that (create a new issue) and I'll happily help (please be sure to ping me in the issue or assign the issue to me so I get notified!

Sorry this happened!

**Comment by sr320** [comment]
@valeste before you delete completely let me archive files for you as I imagine this is only place some files live if they were >100MB. 
Should be done by class time today.
